www/js/lib/util.js

```diff
@@ -194,6 +194,27 @@ define(['q'], function(q) {
     }
     
     /**
+     * Creates a Blob from the given content, then a URL from this Blob
+     * And put this URL in the attribute of the DOM node
```

Comment:
I think we should keep dom and non-dom util functions in different files (to simplify later separation into js-zim and frontend). If they are already mixed, then it is fine.